lib/oauth.js

```diff
@@ -32,11 +31,11 @@ OAuth.prototype.buildParams = function (_oauth, uri, method, query, form, qsLib)
     oa.oauth_signature_method = 'HMAC-SHA1'
   }
 
-  var consumer_secret_or_private_key = oa.oauth_consumer_secret || oa.oauth_private_key
```

Comment:
We shouldn't alter these variable names. They map to actual OAuth standard names so we should keep them.

We should stick `/* eslint-disable camelcase */` and `/* eslint-enable camelcase  */` around the code to just disable this for this code block.